lsstsal/scripts/code/templates/SALDDS.cpp.template

```diff
@@ -350,7 +354,7 @@ salReturn SAL_SALData::salTelemetrySub(int actorIdx)
 
 void SAL_SALData::logError(salReturn status) 
 {
-   cout << "=== ERROR return value = " << status << endl; 
+   cerr << "=== ERROR return value = " << status << endl; 
```

Comment:
logError is used always (at least as far as I can see) with:

```cpp
   if (status != SAL__OK) {
      if (debugLevel >= SAL__LOG_ROUTINES) {
          logError(status);
      }
   }
```

I would prefer tryLogError (or even keep logError..) with the if block replacing the logError calls.